third_party/xla_client/pjrt_computation_client.h

```diff
@@ -126,13 +130,8 @@ class PjRtComputationClient : public ComputationClient {
              std::shared_ptr<PjRtBuffer> buffer)
         : Data(std::move(device), std::move(device_shape)), buffer(buffer) {}
 
-    void* get_handle() const {
```

Comment:
whyb is `get_handle` removed? Is it not being used in any place?

Reply:
`get_handle` was only used by `GetOpaqueHandle` since PJRT doesn't actually have handles and it's not exposed by `Data`. I just rolled it into `GetOpaqueHandle` instead.